Staking FAQ - Wiki Doc (#3101)
w3f/polkadot-wiki: docs/learn/learn-staking-faq.md

---
id: learn-staking-faq
title: Staking FAQ
sidebar_label: Staking FAQ
description: Addresses FAQ on Staking
slug: ../learn-staking-faq
---

# Staking FAQ

## 1. How do I stake my DOT?

Thanks for considering to nominate and stake your DOT to secure the Polkadot Network and its parachains. The success of Nominated Proof of Staking (NPoS) relies on well informed nominators. To directly submit your intent to stake on Polkadot, you should

> -  have an account on a [non-custodial wallet](../build/build-wallets.md) that supports staking. If you are a beginner, watch this tutorial on [creating an account on Polkadot-JS Extension](https://youtu.be/sy7lvAqyzkY).

Reviewer: Section 3 sufficiently covers why someone might get excluded, but in the future I might consider adding another bullet point here along the lines of "make sure you have at least the min-active stake bonded". Although in theory, if there is a huge slashing event this could drop significantly and waiting nominators could suddenly make it in

Reviewer 2: I agree. I will add that info when we create a tutorial showing how to determine that value using the UI

The [staking reward distribution](learn-simple-payouts) is not automatic and needs to be triggered through a transaction. Typically, it is the validators who trigger the staking reward payout transaction, but for some reason if they do not, you can claim the rewards on your own. 

### Nominated validators not in the active set

If none of the validators you nominated made it to the active set in an era, you will not earn any rewards for that era. To maximize your chance of earning staking rewards on every era, consider nominating multiple validators (up to 16 on Polkadot).

### Nominated a validator with 100% commission

When you nominate a validator who takes 100% commission on staking rewards, you will not receive any staking reward for the eras when your nomination is applied to that specific validator.

### Nominated an oversubscribed validator

Only the top 256 nominators of a Validator receive the staking rewards. If your nomination is applied to an oversubscribed Validator and your stake does not put you in the top 256, you will not earn rewards. On a regular basis, ensure that the Validators you chose are not oversubscribed.

### Min-Active Nomination Threshold is not met

> With the [bags-list implementation](https://github.com/paritytech/substrate/pull/9507), the minimum stake threshold to earn staking rewards can change with each era. Do not confuse this with min-intention-threshold (10 DOT) which is required to submit your intention to nominate. The network supports 50,000 nominator intentions, but can reward only up to 22500 nominators in any given era. If you nominate with, say 50 DOT, you are not guaranteed to be part of the top 22500 and hence you may not earn staking rewards. The entry barrier to be an electing nominator is now determined dynamically, based on market demand.

Reviewer: ```suggestion
With the [bags-list implementation](https://github.com/paritytech/substrate/pull/9507), the minimum stake threshold to earn staking rewards can change with each era. Do not confuse this with min-intention-threshold (10 DOT) which is required to submit your intention to nominate. The network supports 50,000 nominator intentions, but can reward at most 22500 nominators in any given era. If you nominate with, say 50 DOT, you are not guaranteed to be part of the top 22500 and hence you may not earn staking rewards. The entry barrier to be an electing nominator is now determined dynamically, based on market demand.
```

The UI will soon hopefully have a min-electing or min-active threshold which will essentially be the lowest staked nominator who is earning. We can then advise that nominators watch this number and target a value above it to increase likelihood of making it into the next election
